pkg/sfu/rtpextension/playoutdelay.go

```diff
@@ -0,0 +1,44 @@
+package rtpextension
+
+import (
+	"encoding/binary"
+	"errors"
+)
+
+const (
+	PlayoutDelayURI = "http://www.webrtc.org/experiments/rtp-hdrext/playout-delay"
+
+	playoutDelayExtensionSize = 3
+)
+
+var (
+	errPlayoutDelayOverflow = errors.New("playout delay overflow")
+	errTooSmall             = errors.New("buffer too small")
+)
+
+//  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+// |  ID   | len=2 |       MIN delay       |       MAX delay       |
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+type PlayOutDelay struct {
+	Min, Max uint16 // delay in 10ms
```

Comment:
okay, made a comment earlier about divide by 10, the explanation is here. Maybe the value can be passed in here as is and divide can happen here?

Reply:
Will think about this. The original point is to keep the extension as same as the document.